### PR TITLE
fix mobile inputs

### PR DESCRIPTION
**Pull request by chrisnojima**
Center inputs on mobile like desktop
@keybase/react-hackers

### CODE REVIEW

File: shared/common-adapters/input.native.js [hunk 1]
```diff
@@ -171,7 +171,8 @@ class Input extends Component<void, Props, State> {
 
     const singlelineStyle = {
       ...commonInputStyle,
-      height: 28,
+      maxHeight: 28,
```
Comment (chromakode) [inline shared/common-adapters/input.native.js]: :sheep: Why was this necessary instead of specifying `height`? Worth explaining in a comment?

Comment (chromakode) [inline shared/common-adapters/input.native.js]: :sheep: Would `_lineHeight` (defined further down) instead of 28 be appropriate here?

Comment (chrisnojima) [inline shared/common-adapters/input.native.js]: using the const and put a comment. If you use height it can still stretch so instead we're using minHeight/maxHeight
